packages/zoe/src/contracts/multipoolAutoswap/multipoolAutoswap.js

```diff
@@ -122,6 +122,16 @@ const start = zcf => {
     return { amountIn, amountOut };
   };
 
+  const getAllPoolBrands = () => {
+    const brands = [];
+    const allBrands = zcf.getTerms().brands;
+    for (const name of Object.getOwnPropertyNames(allBrands)) {
```

Comment:
Better as a filter:
```javascript
    return Object.getOwnPropertyNames(allBrands).filter(name => isSecondary(allBrands[name]);
```
Or if `Object.values` is allowed:
```javascript
    return Object.values(allBrands).filter(isSecondary);
```

Reply:
Not required though.

Reply:
done. Thanks.